docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -79,26 +79,26 @@ H  2.683149  3.028701  3.501215
 
 ### How many images do I need?
 
-It depends... It's usually easier to start with a small number of images and increase if you need to. 
+It depends... It's usually easier to start with a small number of images and increase if you need to. *[A bit vague.]*
```

Comment:
Yes, it's hard to come up to general answers to these kinds of questions...

What I was going for here was that: if you start with loads of images then you might never get close to the converged TS because the optimisation takes too long. If you instead start with a small number and get a reasonable TS structure, you can then use that as your initial intermediate structure in a restarted calculation with more images. Starting small can also quickly give you can rough sense of the energy landscape and that will help determine the actual number of images needed.

Reply:
I would just say this. A tutorial gives you the space to explain the thinking behind doing the calculations well.

Reply:
```suggestion
It depends... It's usually preferable to start with a small number of images and increase the number if you need to. Starting small can quickly give you a rough sense of the energy landscape and that will determine the actual number of images needed. 

In addition, if you start with too many images then you might never get close to the converged TS because the optimisation takes too long. If you instead start with a small number and get a reasonable TS structure, you can then use that as your initial intermediate structure in a restarted calculation with more images.
```